data/json/monstergroups/resort.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "GROUP_RESORT_DANCERS",
+    "type": "monstergroup",
+    "default": "mon_zombie_resort_dancer",
+    "monsters": [
+      { "monster": "mon_zombie_resort_dancer", "freq": 100, "cost_multiplier": 8, "pack_size": [ 6, 10 ] },
+      { "monster": "mon_zombie_resort_bouncer", "freq": 75, "cost_multiplier": 4, "pack_size": [ 1, 3 ] },
+      { "monster": "mon_zombie", "freq": 50, "cost_multiplier": 10, "pack_size": [ 2, 10 ] }
+    ]
+  },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_RESORT_RAID",
```

Comment:
It would guess there is probably already a monster group for police and dog spawns

Reply:
There's GROUP_POLICE with higher cops frequncy compared to basic GROUP_ZOMBIE also GROUP_ZOMBIE_COP which is cops only. I can use second one although it doesn't spawn Z9s.

Reply:
GROUP_ZOMBIE_COP only appears in prisons. There'd be nothing wrong with adding zombie dogs to that group.

Reply:
Same as previous - that's general group so I am careful with changing them.